### PR TITLE
Add mentor star for PR

### PR DESCRIPTION
**Pull request by mariuscoto**

### CODE REVIEW

**Comment by mariuscoto** [comment]
@marianitadn please review

File: views/challenge.jade [hunk 1]
```diff
@@ -111,6 +112,16 @@ block content
...
                       input.btn.btn-success(type='submit', value='Hide')
+
+                  = " Rate "
+                  form(method='post', action='/challenges/' + challenge.link + '/star?id=' + p._id)
+                    select.selectpicker.span1(name='stars', type='text', onchange='this.form.submit()')
+                      each val in [0, 1, 2, 3, 4, 5]
+                        if (p.stars == val)
+                          option(selected)= val
+                        else
+                          option= val
```
Comment (marianitadn) [inline views/challenge.jade]: shouldn't we add a default value of 0 or 1? :)

**Comment by marianitadn** [comment]
look ok, but I would replace `star` with `rating` because stars on GitHub mean something and I find it confusing.

**Comment by marianitadn** [comment]
@mariuscoto I renamed the action and the menu Pull Request section. I will leave this for your review. 
I can't deploy since I have a new RSA key, so you can merge and deploy at will.

**Pull request merged**